buildprogram.bat

```diff
@@ -1,8 +1,13 @@
 
 set mypath=%~dp0
-pyinstaller --onefile resources\twitchcommands.py --add-data "C:\Users\%USERNAME%\AppData\Local\Programs\Python\Python310\Lib\site-packages\ahk";ahk --icon resources\appicon.ico 
+python -c "import sysconfig; print(sysconfig.get_path('purelib') + '\\ahk')" > tmpFile
+set /p PYSITEPKG_DIR= < tmpFile
+DEL tmpFile
+pyinstaller --onefile resources\twitchcommands.py --add-data %PYSITEPKG_DIR%;ahk --icon resources\appicon.ico 
```

Comment:
My site-packages folder was in a different location, this should be more generic - I needed to make a similar change for the launcher [here](https://github.com/Zedb0T/OpenGoal-Mod-Launcher/pull/7/files)